beta/experience-platform-extension/tutorials/README.md

```diff
@@ -25,3 +25,21 @@ Get started with [Assignment 1 - AEP Edge extension setup and XDM usage](https:/
 * [ ] XDM implementation validation
 * [ ] Clarity of documentation during the beta review call
 
+
+### Assignment 2
+
+In this assignment you will learn how to:
+
+1. Install Swfit Core library using Cocoapods
+2. Initialize SDK with new API
+3. Start/Stop lifecycle
+4. Make an Identity call
+
+Get started with [Assignment 1 - AEP Edge extension setup and XDM usage](https://aep-sdks.gitbook.io/docs/beta/experience-platform-extension/tutorials/tutorial-1-edge-extension-setup).
+
+#### Prepare to provide feedback on:
+
+* [ ] Overall setup and implementation workflow of AEPCore SDK
+* [ ] In Pod file, instead of just include ACPCore, now you need to cinlude AEPServcies, AEPCore, AEPLifecyle, AEPIdentity, AEPSignal, does that bother you?
```

Comment:
`instead of just including ACPCore, you now need to include AEPCore, AEPServices, AEPLifecycle, AEPIdentity, and AEPSignal. This allows greater flexibility for both developers and consumers of the SDK to only include what they need. Do you feel the trade-off is an overall benefit?`